src/generators/static-theme.ts

```diff
@@ -60,7 +60,11 @@ function mix(color1: number[], color2: number[], t: number) {
 export default function createStaticStylesheet(config: FilterConfig, url: string, frameURL: string, staticThemes: StaticTheme[]) {
     const srcTheme = config.mode === 1 ? darkTheme : lightTheme;
     const theme = Object.entries(srcTheme).reduce((t, [prop, color]) => {
+        const [r, g, b, a] = color;
         t[prop] = applyColorMatrix(color, createFilterMatrix({...config, mode: 0}));
+        if(a !== undefined) {
+            t[prop].push(a);
+        }
```

Comment:
```suggestion
        a && t[prop].push(a);
```

Welcome to modern javascript whereby it will be automatically checked for truthiness 😄

Reply:
Alpha value could be zero I guess? If at some point someone wants to represent the color "transparent".

Reply:
That's correct... I'm not sure how dynamic is handling as in there case it seems `a` is always defined.

Reply:
As if it's defined, the check could be removed altogether.

Reply:
Nevermind about what I said. This code is good to go. Il add unit tests for this one later in another PR.